packages/web-app-files/src/components/SideBar/Links/PublicLinks/LinkEdit.vue

```diff
@@ -100,7 +100,7 @@
           </oc-button>
         </div>
         <div>
-          <oc-button v-if="saving" variation="primary" appearance="filled" disabled>
+          <oc-button v-if="saving" variation="primary" appearance="filled" disabled class="oc-saving-public-link-button">
```

Comment:
`id` would be greate here
```suggestion
          <oc-button v-if="saving" variation="primary" appearance="filled" disabled id="oc-saving-public-link-button">
```